README.md

```diff
@@ -203,7 +203,11 @@ There are some rules for prefixing to observe:
 
 `Sanitizer`:
 
-* Determines prefix as any text (including whitespace) that comes before `REPOBEE-SANITIZER-START`
+* Determines prefix as any text before `REPOBEE-SANITIZER-START`
+  * This excludes any whitespace before the `START` marker
+    * Pro-Tip: There can be any amount of whitespace on any line using a prefix, see figure 4. This allows easy alignment of code
```

Comment:
I quite like having this as a "pro-tip" as it draws the eye of the reader towards what it really is, an explanation of a neat feature